CLI ready for review

georeri opened the pull request:
CLI is cleaned up, using all system operations, and is ready for a review to see if we need to adjust anything before mirroring in dotnet and python.

Commit:
CLI wrapped around system ops

Commit:
Cleaned up CLI prompt

Commit:
Fixing java version

Commit:
Removing java version from build.gradle

Commit:
Made move commands only work if the game is started.

jpwhite3 committed:
Fixing the run target in the makefile

Commit:
Updated to make cli clearer

jpwhite3 commented on src/main/java/com/levelup/forestsandmonsters/cli/LevelUpGame.java:
```diff
@@ -0,0 +1,114 @@
+  return !isGameStarted
+  ? Availability.available()
+  : Availability.unavailable("game already started");
+}
```
The indentation of this section (lines 30-58) made it hard to read, but I don't know the Java formatting conventions.

Commit:
Standarized format on save. Minor readme updates.

Commit:
Standardized format on save

Commit:
CLI updated and syncd with dotnet

Commit:
Fixing missing updates from refactoring in VSCode

Commit:
Attempting test result publish fix

Commit:
Fixed the feature file. VSCode formatting was adding whitespace.

georeri commented:
@jpwhite3 I didn't realize the pull request was still open, so extra commits are here. This one is in sync with the dotnet changes. I also fixed a few problems with publishing test results, since we added the results folders to gitignore. Should be good to go.
